app/javascript/partials/_nested_forms.js

```diff
@@ -83,6 +88,38 @@ $(function() {
     ...defaultNestedFormsOptions,
     beforeAddForm: ($container) => {
       $container.children('.nested-medium-form').removeClass('d-none');
+    },
+    afterAddForm: function(_, $form) {
+
+      // If we only have one nested media form, we don't have to do anything
+      // becasue the index of 0 is already the right one
+      if ($('.nested-medium-form').length === 1) {
+        return;
+      }
+
+      // Increment the index on all elements of the new form by 100
```

Comment:
how come 100? and where is this represented in code? i cannot find something about incrementing by 100

Reply:
will be adjusted and refactored in #193